LiskHQ/lisk-service
# Update DPoS Received Votes endpoint

### What was the problem?
This PR resolves #1349 

### How was it solved?
- [x] The endpoints are updated as expected above
- [x] The response body is up-to-date
- [x] All the methods, variables, files are renamed and refactored appropriately
- [x] Swagger is up-to-date
- [x] Integration tests are up-to-date 

### How was it tested?
- Run Lisk Service against Lisk Core v4.0.0-alpha.5
- Add dummy data entries to the `stakes` table and call the endpoint
- Copy swagger from the `/spec` response and check on editor.swagger.io
